devs/run-tests/10encodeURIComponent.ts

```diff
@@ -0,0 +1,35 @@
+import { assert } from "@devicescript/core"
+import {
+    encodeURIComponent,
+} from "@devicescript/core/src/encodeURIComponent"
```

Comment:
Is this the same situation as the Map class where it needs to be in runtime? Or is this the correct approach for a export function?

Reply:
yes it is the same situtation. I'll chat with Michal about this. We might want to put them in runtime in the meanwhile until get things figured out. You're faster than us!

Reply:
Enjoying the challenges great way to break up the work day.

Reply:
Ok, I moved the function to runtime and moved tests as well, but I see the submodule issues with jacdac again. If there's anything I'm doing to cause this feel free to let me know hate having such bumpy PR's!

Reply:
we'll fix the submodule (and we should write a guide bout those)

Reply:
Let me know if i can help.